### ISSUE TITLE
Installation of python2 triggered on openSUSE Leap 15.x even though python3 is installed

### ISSUE THREAD

**Issue by johanneskastl** [ceph/ceph-ansible#4325]
**Bug Report**

What happened:
ceph-ansible installs python2 on openSUSE 15.x, even if python3 is already installed and fully working.

I guess this is due to the following `when`-condition not detecting the python3 installation as this only provides `/usr/bin/python3` , not `/usr/bin/python`:
```
- name: check for python
  stat:
    path: /usr/bin/python
  ignore_errors: yes
  register: systempython
[...]
  when: systempython.stat is undefined or not systempython.stat.exists
```

What you expected to happen:
python2 is not needed and should not be installed.

(Installation of python-xml is currently safeguarded in PR #4324, but that is a later task) 

How to reproduce it (minimal and precise):
Run ceph-ansible on openSUSE Leap 15.x hosts. Watch for the output of the tasks from [raw_install_python.yml](https://github.com/ceph/ceph-ansible/blob/master/raw_install_python.yml)

**Environment**:
* OS (e.g. from /etc/os-release):
```
NAME="openSUSE Leap"
VERSION="15.1 "
ID="opensuse-leap"
ID_LIKE="suse opensuse"
VERSION_ID="15.1"
PRETTY_NAME="openSUSE Leap 15.1"
ANSI_COLOR="0;32"
CPE_NAME="cpe:/o:opensuse:leap:15.1"
BUG_REPORT_URL="https://bugs.opensuse.org"
HOME_URL="https://www.opensuse.org/"
```
* Kernel (e.g. `uname -a`):
```
Linux XXX 4.12.14-lp151.28.10-default #1 SMP Sat Jul 13 17:59:31 UTC 2019 (0ab03b7) x86_64 x86_64 x86_64 GNU/Linux
```
* Ansible version (e.g. `ansible-playbook --version`):
```
ansible 2.8.2
  config file = /home/tux/ceph-ansible/ansible.cfg
  configured module search path = ['/home/tux/ceph-ansible/library']
  ansible python module location = /usr/lib/python3.6/site-packages/ansible
  executable location = /usr/bin/ansible
  python version = 3.6.5 (default, Mar 31 2018, 19:45:04) [GCC]
```
* ceph-ansible version (e.g. `git head or tag or stable branch`):
```
master at 2fdf7316a41e2bb5c331cca9a3e375a09700d194
stable-4.0 at 0f90ffe9dfa312b2c8caf93bd6e0c0b9c25021b4
```

**Comment by johanneskastl** [comment]
Might be solved by [this commit](https://github.com/ceph/ceph-ansible/pull/4296/commits/de20df03b4f807fc2f1cd9d65df6658da2161db7), will check.

**Comment by johanneskastl** [comment]
Solved by #4351
